config/config.go

```diff
@@ -84,19 +101,23 @@ type BlockSubmitterConfig struct {
 	// Denomination is the token denomination of the celestia chain being used
 	// for a data availability layer. Defaults to "tia"
 	Denom string `toml:"denomination"`
-
-	// RPCAddress is the rpc address to submit celestia transactions to
-	RPCAddress string `toml:"celestia-rpc-addr"`
+	// RPCAddress is the rpc address to submit celestia transactions to and
+	// become a light client of. Defaults to "127.0.0.1:9090"
+	GRPCAddress string `toml:"celestia-grpc-addr"`
```

Comment:
this is a simple name change, but hopefully it helps differentiate this address from the rest api address